tests/regression/lp-1848567/task.yaml

```diff
@@ -0,0 +1,44 @@
+summary: ensure apparmor_parser is not using gobs of memory
+details: |
+  Ensure that apparmor_parser is not using excessive amounts of memory when
+  compiling snap-update-ns profile of an application that looks like
+  gnome-system-monitor snap.
+prepare: |
+  # Pack, install and connect the three participants.
+  snap pack test-snapd-app
+  snap pack test-snapd-gnome-3-28-1804
+  snap pack test-snapd-gtk-common-themes
+
+  snap install --dangerous test-snapd-app_1_all.snap
+  snap install --dangerous test-snapd-gnome-3-28-1804_1_all.snap
+  snap install --dangerous test-snapd-gtk-common-themes_1_all.snap
+
+  snap connect test-snapd-app:gnome-3-28-1804 test-snapd-gnome-3-28-1804
+  snap connect test-snapd-app:icon-themes test-snapd-gtk-common-themes:icon-themes
+  snap connect test-snapd-app:sound-themes test-snapd-gtk-common-themes:sound-themes
+execute: |
+  # TODO: expand this test to install GNU time or find a suitable replacement.
+  if snap debug sandbox-features --required apparmor:kernel:mount && [ "$(command -v time)" != time ]; then
```

Comment:
As Samuele pointed out in SU, perhaps you could use `/usr/bin/time` from the base snaps, but note that I don't actually see `/usr/bin/time` in core or core18, so you would have to get it added there first.

Reply:
Ohh, drat. I think I will just NIH a time-tool that does the equivalent without being a pain to use over time.

Reply:
This is a very nice test, but I think it's also ok to land it as a followup (given the annoying issue with time).

Reply:
The test works as-is on some systems so it's already valuable as-is. We can iterate on it after the bug fix is merged so that it is tested more broadly on other systems.

Reply:
With hello-world.sh, I noticed that `command -v time` is going to give `time` on UC16 (at least), but `command time -f "%M" ...` will fail with "bash: time: command not found". If in hello-world.sh I use /bin/sh (dash), then `command -v time` doesn't display 'time'. I mention this cause this seems to be working due to use of dash, but it seems brittle.

Furthermore, I don't care for the time check since it means we'll get a pass if the time check shortcircuits everywhere and that would go unnoticed. Normally we use `systems` to set the expectations. Why can't we do something similar here?